azdev/utilities/command.py

```diff
@@ -31,10 +32,16 @@ def call(command, **kwargs):
     :param kwargs: Any kwargs supported by subprocess.Popen
     :returns: (int) process exit code.
     """
-    return subprocess.call(
-        command,
-        shell=True,
-        **kwargs)
+    from azdev.utilities import IS_WINDOWS
+    cmd_args = command
+    if IS_WINDOWS and command.startswith('az '):
+        cmd_args = "az.bat " + command[3:]
+    if not IS_WINDOWS:
```

Comment:
Consider using `else`.

Reply:
Actually it would be `elif`.

These two branches deal with different type of situations, the first one is correcting `az` cmd in windows, the second one is correcting arg type in linux-like system, so use two `if`, not `if elif` without `else` here.